### PR TITLE
Setup logging

### LINKED ISSUE #17: Add proper logger
- add logger with levels

### PR DESCRIPTION
**Pull request by MakMuftic**
### Changes

- Added _logrus_ logger
- Refactored logging through code

### Issues

closes #17

### CODE REVIEW

**Review by mpetrun5 — DISMISSED** [review]

File: internal/controllers/metrics.go [hunk 1]
```diff
@@ -45,8 +46,10 @@ func (c ApiController) SaveMetricsHandler(w http.ResponseWriter, r *http.Request
 
 	if err != nil {
 		// error on saving in database
-		log.Println(err.Error())
+		log.Errorf("Unable to save metrics for node %v to database, error: %v", requestContext.NodeId, err)
 		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 		return
 	}
+
+	log.Debugf("Node %s saved new metrics", requestContext.NodeId)
```
Comment (mpetrun5) [inline internal/controllers/metrics.go]: Maybe add saved metrics

File: internal/controllers/metrics.go [hunk 2]
```diff
@@ -45,8 +46,10 @@ func (c ApiController) SaveMetricsHandler(w http.ResponseWriter, r *http.Request
 
 	if err != nil {
 		// error on saving in database
```
Comment (mpetrun5) [inline internal/controllers/metrics.go]: Probably redundant as you log the same thing the line below it

File: internal/controllers/metrics.go [hunk 3]
```diff
@@ -24,10 +24,11 @@ func (c ApiController) SaveMetricsHandler(w http.ResponseWriter, r *http.Request
 		var mr *util.MalformedRequest
 		if errors.As(err, &mr) {
 			// malformed request error
```
Comment (mpetrun5) [inline internal/controllers/metrics.go]: Same

File: internal/controllers/metrics.go (hunk 1 above)
Comment (mpetrunic) [reply internal/controllers/metrics.go]: just important ones like finalized and head block slot

**Review by mpetrun5 — APPROVED** [review]

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**